psuedo-icecream.txt

```diff
@@ -0,0 +1,12 @@
+Get all flavors     =>   "/flavors"
+Get specific flavor =>   "/flavors/:types"
+
+Get all categories  =>   "/categories"
+Get specific category => "/categories/:types"
```

Comment:
Again, I'd suggest "/categories/:id" or "/categories/:name" (the name of the category) to be more clear.